internal/storage/rocksdb/store.go

```diff
@@ -508,34 +662,83 @@ func toByteArray(value *gorocksdb.Slice) []byte {
 	return res
 }
 
+func parseTTLMsgPackData(valueWithTtl []byte) (*ttlDataFormat, error) {
+	var row ttlDataFormat
+	var err error
+	if len(valueWithTtl) > 0 {
```

Comment:
If this check is dropped, check if the unmarshal handles this gracefully and returns the error, instead of crashing. In that case, we can drop this check. Please verify.

Reply:
unmarshal doesn't handle that, that's why I added this check.